src/connections/Task.ts

```diff
@@ -173,12 +154,17 @@ const taskConnection: Connection<TaskConnection, TaskProps> = {
             taskIsUpdating,
             taskUpdateFailure,
 
-            submitForApproval: ({ siteReportNothingToReportUuid, nurseryReportNothingToReportUuid, feedback }) =>
+            submitForApproval: () => updateTask(taskResponse.attributes.uuid, { status: "awaiting-approval" }),
+
+            bulkApproveReports: (
+              feedback?: string,
+              siteReportNothingToReportUuids?: string[],
+              nurseryReportNothingToReportUuids?: string[]
+            ) =>
               updateTask(taskResponse.attributes.uuid, {
-                status: "awaiting-approval",
-                siteReportNothingToReportUuid: siteReportNothingToReportUuid ?? null,
-                nurseryReportNothingToReportUuid: nurseryReportNothingToReportUuid ?? null,
-                feedback: feedback ?? null
+                feedback: feedback,
+                siteReportNothingToReportUuids: siteReportNothingToReportUuids,
+                nurseryReportNothingToReportUuids: nurseryReportNothingToReportUuids
```

Comment:
when assigning properties, if it's using the same name you don't have to us the colon syntax. That's why my original suggestion for this change used:
```
{ feedback, siteReportNothingToReportUuids, nurseryReportNothingToReportUuids }
```